packages/cli/src/backups/index.ts

```diff
@@ -104,9 +108,9 @@ async function importBackup(opts: BackupOpts) {
 
 async function pickOne(opts: BackupOpts) {
   if (opts["import"]) {
-    return importBackup(opts)
+    return await importBackup(opts)
```

Comment:
NAB: Just letting you know, this isn't technically necessary - doing this means that error handling happens within this function so:
```js
async function something() {
  try {
    return someOtherAsyncFunction()
  } catch (err)...
}
```
Versus:
```js
async function something() {
  try {
    return await someOtherAsyncFunction()
  } catch (err)...
}
```
The first function the try catch will never be triggered because the calling function handles any errors that erupt from the `someOtherAsyncFunction` but in the second example the error will be caught within the function. If the calls are not wrapped in any kind of error handling then it is going to bubble up anyway and the await does nothing.